capellambse/svg/drawing.py

```diff
@@ -66,7 +71,8 @@ def __init__(self, metadata: generate.DiagramMetadata):
         self.__drawing = drawing.Drawing(**superparams)
         self.diagram_class = metadata.class_
         self.deco_cache: set[str] = set()
-        self._add_backdrop(pos=metadata.pos, size=metadata.size)
+        if not transparent_background:
+            self._add_backdrop(pos=metadata.pos, size=metadata.size)
```

Comment:
It's a way to get a transparent background, which I want to use in the context-diagrams documentation. But I did disable it now b/c I also need complementary colors for the dark mode in order to make text readable again. The implementation also allows the propagation of parameters from the JSON to the drawing. So finally font-family and font-size are configurable too.

Reply:
What do we do about pre-rendered / cached diagrams though? We don't have any way to change stuff like fonts in Capella diagrams (without completely messing up the entire layout). I suppose we could remove the backdrop rect there, but I still don't really think it's a particularly great idea to be honest. And you even found the issue yourself immediately 🙃 

> I also need complementary colors for the dark mode in order to make text readable

This is exactly the reason why the white backdrop exists. My vote goes for keeping it in and keeping our code simple.

Talking about dark mode, I have quite good experiences with simply inverting the brightness values if I don't have full control over the color palette of an image. This is a CSS filter chain I have in active use and which serves me well:

```css
filter: invert() hue-rotate(180deg) brightness(0.8)
```

You might want to experiment a bit with this or similar filters, this might completely eliminate the need for customizing the backdrop.

Reply:
Look. There are many ways to achieve what I am after. I can parse the SVG and remove the backdrop rectangle... but since I know where it gets added, I wanted to make it configurable. I don't see how this makes the super simple code of initializing a Drawing more complex. Let's keep the church in the village.
When I try to render a diagram from the diagram cache (Capella made) as an `SVGDiagram` I get a RuntimeError: `Diagram not in cache:`. So no issues with changing font-family or font-size.